### ISSUE TITLE
Gitlab config docs missing

### ISSUE THREAD

**Issue by ajberasategui** [danger/danger-js#927]
Documentation on how to configure Danger JS for Gitlab is missing on the site.

**Comment by NarHakobyan** [comment]
yeah I would try this with gitlab

**Comment by sluongng** [comment]
Yup, some documentation and perhaps examples would go a long way for both gitlab `platform` and `CI`
